projects/core/src/navigation/navigation.element.ts

```diff
@@ -210,27 +210,72 @@ export class CdsNavigation extends LitElement {
     return Array.from(this.allNavigationElements).filter(n => isVisible(n));
   }
 
-  addStartEventListener() {
```

Comment:
I reordered some of the functions here to group lifecycle events together

Reply:
Can you reorder the these in a separate commit to make the changes easier to review?

Reply:
Sure, updated.

Reply:
Thanks! I think you meant navigation component, not range component, in the commit message. You can also just use the commit scope. "chore(navigation): reorder lifecycle events for readability"